packages/ses/src/compartment-shim.js

```diff
@@ -141,6 +166,11 @@ export class Compartment {
       instances,
       globalTransforms,
       globalObject,
+      // The caller continues to own the globalLexicals object they passed to
+      // the compartment constructor, but the compartment only respects the
+      // original values and they are constants in the scope of evaluated
+      // programs and executed modules.
+      globalLexicals: freeze({ ...globalLexicals }),
```

Comment:
Note that only the value (not the descriptor) of only enumerated own properties are sampled.

Reply:
Please add something like this note to the comment here.